Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUCompositorIntegrationImpl.h

```diff
@@ -86,12 +86,14 @@ class CompositorIntegrationImpl final : public CompositorIntegration {
 
     Vector<RetainPtr<IOSurfaceRef>> m_renderBuffers;
     WTF::Function<void(CFArrayRef)> m_renderBuffersWereRecreatedCallback;
+    void setWebProcessID(unsigned) final;
 #endif
 
     WTF::Function<void(CompletionHandler<void()>&&)> m_onSubmittedWorkScheduledCallback;
 
     RefPtr<PresentationContextImpl> m_presentationContext;
     Ref<ConvertToBackingContext> m_convertToBackingContext;
+    unsigned m_webProcessID { 0 };
```

Comment:
Isn't there a `mach_foo_t` type we can use instead of unsigned?

Reply:
There is `mach_port_t` which is typedef as `unsigned int` but I don't think that is available in PAL. I could wrap all the usage of it in `#ifdef` however but this defeats the purpose of having a PAL 

I'm going to see if I can refactor some of these changes to have minimal impact on PAL.

Reply:
I switched to `mach_port_t` as most PAL changes were removed. I think it will be fine, if there is a build issue on some platform, I will add a typedef